### PR TITLE
React Router Dom added

### PR DESCRIPTION
**Pull request by yasincaliskan**
I added `react-router-dom` package. When your enhancements complete, I will implement router Hooks. I will pull your commits and continue with this PR.

### CODE REVIEW

File: src/app/timeline/TimeLine.tsx [hunk 1]
```diff
@@ -1,11 +1,11 @@
 import React from 'react'
-import { Survey, SurveyTimeLine } from '../components/survey/Survey'
+import { Survey, SurveyTimeLine } from '../../components/survey/Survey'
 
 interface TimeLine {
   surveyList: Survey[];
 }
 
-const mockSurveys = require("../mockSurveyData.json");
+const mockSurveys = require("../../mockSurveyData.json");
```
Comment (bilginyuksel) [inline src/app/timeline/TimeLine.tsx]: please revert this change this will lead to conflict this branches only purpose is to create routing mechanism.

**Review by bilginyuksel — COMMENTED** [review]
I think we need to wait to merge this pull request because it is related to others.

**Comment by yasincaliskan** [comment]
mockSurveyData.json change reverted. By the way, you should run `npm install` for install `react-router-dom`.

**Review by HasancanKazan — APPROVED** [review]
